tests/gcp/operators/test_bigquery.py

```diff
@@ -566,3 +566,26 @@ def test_execute(self, mock_hook):
                 deletion_dataset_table=deletion_dataset_table,
                 ignore_if_missing=ignore_if_missing
             )
+
+
+class TestBigQueryGetDatasetTablesOperator(unittest.TestCase):
+    @mock.patch('airflow.gcp.operators.bigquery.BigQueryHook')
+    def test_execute(self, mock_hook):
+        operator = BigQueryGetDatasetTablesOperator(
+            task_id=TASK_ID,
+            dataset_id=TEST_DATASET,
+            project_id=TEST_GCP_PROJECT_ID,
+            max_results=2
+        )
+
+        operator.execute(None)
```

Comment:
```suggestion
        operator.execute(context=None)
```
just my preference - change it only if you agree :)